### PR TITLE
[camera] Saving response - video type into realm database.

### PR DESCRIPTION
**Pull request by DevanshuQuovantis**
Added realm integration.
Saving video response into realm.
Audio recording component design.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -134,6 +134,8 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
+        multiDexEnabled true
+        missingDimensionStrategy 'react-native-camera', 'general' // <--- insert this line
```
Comment (ShikharQuovantis) [inline android/app/build.gradle:138]: this is for what purpose?

File: ios/Assignment.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```
Comment (ShikharQuovantis) [inline ios/Assignment.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:5]: again? for what purpose?

**Review by ShikharQuovantis — CHANGES_REQUESTED** [review]

File: android/app/build.gradle (hunk 1 above)
Comment (DevanshuQuovantis) [reply android/app/build.gradle:138]: As per the [documentation](https://github.com/react-native-camera/react-native-camera/blob/master/docs/installation.md#requirements), we need to add missingDimensionStrategy.
I tired without adding missingDimensionStrategy, but then library for camera failed to initialised.
For multiDex, while building app in Android, an error occurred stating [this](https://stackoverflow.com/questions/26633591/multidex-support-in-android-application-error)

File: ios/Assignment.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 2 above)
Comment (DevanshuQuovantis) [reply ios/Assignment.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:5]: For this, I haven't changed any of the native files/workspace/pbxproj files except info.plist where I added permissions.
I think, this is generated automatically by Xcode.

**Review by ShikharQuovantis — APPROVED** [review]

**Pull request merged**